platform/android/java/lib/src/org/godotengine/godot/plugin/GodotPlugin.java

```diff
@@ -30,6 +30,7 @@
 
 package org.godotengine.godot.plugin;
 
+import android.os.Bundle;
```

Comment:
This should go in the `android.*` section below.
https://travis-ci.org/github/godotengine/godot/jobs/698705491